web/themes/new_weather_theme/assets/sass/components/radar.scss

```diff
@@ -1,12 +1,57 @@
 @use "uswds-core" as *;
 
 .wx-radar-container {
-  aspect-ratio: 3/4;
+  aspect-ratio: 3/2;
```

Comment:
The design calls for a starting aspect ratio of `6/4` on mobile and `3/2` on tablet+, but AFAIK those are the same number. 😆